src/services/requestHandler.js

```diff
@@ -10,8 +10,14 @@ const axiosInstance = () => {
   if (process.env.NODE_ENV === 'development') {
     apiUrl = 'http://localhost:3000/api';
   }
+
+  let token = '';
+  if (localStorage.getItem('token')) {
```

Comment:
I don't think this approach is ok because
- it forces everyone to use localStorage
- it forces everyone to one token source which can lead to conflict (e.g the reset password token is usually different from the login token.

My suggested approach will be to pass the token as an argument and set a default value of empty string so that anyone that doesn't need it does not have to pass anything.
Something like:
```
const axiosInstance = (token = '') => {
  let apiUrl = 'https://authorhaven.herokuapp.com/api';
  if (process.env.NODE_ENV === 'development') {
    apiUrl = 'http://localhost:3000/api';
  }
  const instanceCreate = axios.create({
    baseURL: apiUrl,
    headers: {
      'Content-Type': 'application/json',
      Authorization: token,
    },
  });
  return instanceCreate;
};
```
Note that you don't have to explicitly create an instance of the `Headers` class.